seedfarmer/cli_groups/_list_group.py

```diff
@@ -779,8 +779,20 @@ def list_build_env_params(
 @list.command(
     name="schema",
     help="""Generate the schema that SeedFarmer uses for manifest objects.
+             Either the deployment manifest or module manifest schema
+             can be requested.
              This will return a formatted string of the schema that can
              be piped to a file. """,
 )
-def list_manifest_schema() -> None:
-    print(json.dumps(bi.get_manifest_schema(), indent=2))
+@click.option(
+    "--type",
+    "-t",
+    help="Either 'deployment' or 'module' can be used, default is `deployment`",
+    required=False,
+    default="deployment",
+)
+def list_manifest_schema(type: str) -> None:
+    if type not in ["deployment", "module"]:
+        print_bolded(message="Only 'deployment' or 'module' can be passed in", color="yellow")
```

Comment:
We can use [click.Choice](https://click.palletsprojects.com/en/8.1.x/options/#choice-options) to perform this validation for us.

Reply:
Will research